packages/traceability/issuer-api/src/pods/certificate/handlers/sync-certificate.handler.ts

```diff
@@ -27,6 +27,8 @@ export class SyncCertificateHandler implements IEventHandler<SyncCertificateEven
         try {
             await certificate.sync();
         } catch (e) {
+            this.logger.error(`Failed to resync certificate: ${JSON.stringify(e.message)}`);
```

Comment:
Don't we need information about tokenId, so we can debug which particullary certificate failed?
We have no guarantee, that such information will be in `e.message`, haven't we?

We can relate this with the `.log` in the beginning of this function, but that may be tedious to do.

Reply:
The error will probably include the tokenId + it's logged in the beginning of the function, but you're right - would be better to have it in the same log itself.